### DISCUSSION TITLE
Previous QSOs with Callsign

### DISCUSSION THREAD

**Discussion by M0TZO** [magicbug/Cloudlog#623] [Ideas / Feature Requests]
|In the the pop up QSO details box, would it be possible to show Previous QSOs?

Also on the Homepage &/or Logbook pages maybe a symbol (e.g. 2M0SQL + ) to show there are previous QSOs?

I've worked a few recently that Gridtracker is saying Previous bands / modes worked,

**Comment by magicbug** [comment]
I've been thinking about a More Details full page for worked callsigns that would probably be the better solution rather than adding it to the QSO details popup.


This would show everything in the system about the callsign, past QSOs, QSL statuses etc.

**Comment by magicbug** [comment]
Currently won't be added any time in the next couple of weeks, I need to get my focus back on pdf label generation for a bit.
